cmd/dtab.go

```diff
@@ -93,7 +107,7 @@ var (
 				if err != nil {
 					return err
 				}
-				_, err = ctl.Create(name, dtabstr)
+				_, err = ctl.Create(name, dtabstr, dtabJson)
```

Comment:
The problem is I forgot to update these commands to support the `{"dtab": ..., "version" ...}` structure (whereas using the dtab array directly works fine). I will post a fix thank you!